index.js

```diff
@@ -126,7 +126,23 @@ class MbusMaster {
         });
     }
 
-    getData(address, callback) {
+    getData(address, options, callback) {
+        // default options
+        // pingFirst: Work-around buggy behaviour with some M-Bus devices,
+        //            notably Sontex Supercal531
+        //            https://github.com/rscada/libmbus/pull/95
+        let pingFirst = true;
+
+        if (typeof(options) === "function") {
+            callback = options;
+            options = null;
+        }
+
+        if (options) {
+            // de-structure
+            ({pingFirst} = options);
```

Comment:
I think if `pingFirst` is not given in the options here, then it would be overwritten as `undefined` – is that the intended behavior?

Reply:
correct ... I will fix this after merge